package.json

```diff
@@ -101,6 +101,7 @@
         "eslint-plugin-prettier": "^4.0.0",
         "eslint-plugin-react": "^7.28.0",
         "eslint-plugin-react-hooks": "^4.3.0",
+        "immer": "^9.0.15",
```

Comment:
why is immer added explicitly? I don't see it directly used.

Reply:
There was some type error in redux toolkit and I found some issue that suggested to explicitely add immer to deps. Now I tried to remove it and it seems to work.